system-test/reader_client_test.ts

```diff
@@ -97,9 +124,26 @@ describe('reader.ReaderClient', () => {
       .dataset(datasetId)
       .table(tableId)
       .insert([
-        {name: 'Ada Lovelace', row_num: 1},
-        {name: 'Alan Turing', row_num: 2},
-        {name: 'Bell', row_num: 3},
+        {
+          name: 'Ada Lovelace',
+          row_num: 1,
+          optional: 'Some data',
+          list: [1],
+          metadata: {
+            created_at: bigquery.timestamp('2020-04-27T18:07:25.356Z'),
+            updated_at: bigquery.timestamp('2020-04-27T20:07:25.356Z'),
```

Comment:
```suggestion
            updated_at: bigquery.timestamp('2020-04-27T20:07:35.356Z'),
```

Reply:
even if it doesn't change things, let's add 10 min for posterity 😁

Reply:
I've already pushed a change that adds 2 hours